platforms/documentation/docs/src/docs/userguide/dep-man/declaring-repositories/supported_metadata_formats.adoc

```diff
@@ -0,0 +1,162 @@
+5. **Capabilities**: Describes the functionality or features that a module provides or consumes, helping to avoid conflicts when different modules provide the same capability.
+
+This metadata is typically stored in files like `pom.xml` for Maven or `ivy.xml` for Ivy, and Gradle can also generate and consume metadata in its own format, called Gradle Module Metadata (GMM).
+This metadata plays a crucial role in dependency resolution, ensuring that the right versions and variants of modules are used in a build.
+
+[[sec:supported-metadata-formats]]
+== Supported metadata formats
+
+External module dependencies require module metadata so that Gradle can determine the transitive dependencies of a module.
+Gradle supports various metadata formats to achieve this.
+
+[[sub:supported-md-gradle]]
+=== Gradle Module Metadata (GMM) files
+
+Gradle Module Metadata is specifically designed to support all features of Gradle's dependency management model, making it the preferred format.
+
+You can find the https://github.com/gradle/gradle/blob/master/platforms/documentation/docs/src/docs/design/gradle-module-metadata-latest-specification.md[specification here].
+
+[source,json]
+----
+{
+  "formatVersion": "1.1",
+  "component": {
+    "group": "com.example",
+    "module": "my-library",
+    "version": "1.0"
+  }
+}
+----
+
+[[sub:supported-md-pom]]
+=== POM files
+
+Gradle natively supports https://maven.apache.org/pom.html[Maven POM files].
+By default, Gradle will first look for a POM file.
+However, if the file contains a special marker, Gradle will use <<#sub:supported-md-gradle, Gradle Module Metadata>> instead.
+
+[source,xml]
+----
+<project xmlns="http://maven.apache.org/POM/4.0.0">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example</groupId>
+  <artifactId>my-library</artifactId>
+  <version>1.0</version>
+</project>
+----
+
+[[sub:supported-md-ivy]]
+=== Ivy files
+
+Gradle also supports http://ant.apache.org/ivy/[Apache Ivy metadata files].
+Gradle will first look for an `ivy.xml` file, but if this file contains a special marker, it will use <<#sub:supported-md-gradle, Gradle Module Metadata>> instead.
+
+[source,xml]
+----
+<ivy-module version="2.0">
+  <info organisation="com.example" module="my-library" revision="1.0"/>
+  <dependencies>
+    <dependency org="org.example" name="dependency" rev="1.2"/>
+  </dependencies>
+</ivy-module>
+----
+
+[[sec:supported_metadata_sources]]
+== Supported metadata sources
+
+When searching for a module in a repository, Gradle checks for supported metadata file formats by default.
+
+In a Maven repository, Gradle looks for `.pom` files.
+In an Ivy repository, it checks for `ivy.xml` files.
+And in a flat directory repository, it looks directly for `.jar` files without expecting any metadata.
+Starting with Gradle 5.0, Gradle also looks for `.module` (Gradle module metadata) files.
```

Comment:
I don't think we need to say "Staring in Gradle 5.0". This has been the default for a while.

I also think we should order this list in: GMM, POM, Ivy, Flat

Reply:
Done.